pandas/io/pytables.py

```diff
@@ -389,6 +389,10 @@ def __init__(self, path, mode=None, complevel=None, complib=None,
         except ImportError as ex:  # pragma: no cover
             raise ImportError('HDFStore requires PyTables, "{ex}" problem importing'.format(ex=str(ex)))
 
+        if complib not in (None, 'blosc', 'bzip2', 'lzo', 'zlib'):
```

Comment:
but then reading over the old issue. I am not sure these ``blosc`` compressors work (though not entirely sure why)